emrun.py

```diff
@@ -1605,9 +1612,9 @@ def run():
 
   if not options.serve and (options.system_info or options.browser_info):
     # Don't run if only --system_info or --browser_info was passed.
-    options.no_server = options.no_browser = True
+    options.run_server = options.run_browser = False
 
-  if not options.serve and not (options.no_server and options.no_browser):
+  if not options.serve and not options.run_server and not options.run_browser:
```

Comment:
this was `!x && !(!a && !b)` (where `a == server` etc.) that should become `!x && (a || b)` I think?

Reply:
Well spotted!